### PR TITLE
Add jtreg jcov settings for Test coverage

### PR DESCRIPTION
**Pull request by neomatrix369**
[DONT MERGE TILL PR #20] has been merged

Also test it out in Jenkins via en existing job. 

Related to issue #7 - partially/fully resolves it.

### CODE REVIEW

File: sbin/build.sh [hunk 1]
```diff
@@ -200,7 +201,7 @@ echo "Building the JDK: calling $makeCMD"
 $makeCMD
 
 if [ $? -ne 0 ]; then
-  echo "${fail}Failed to make the JDK, exiting"
+   echo "${fail}Failed to make the JDK, exiting"
   exit;
```
Comment (gdams) [inline sbin/build.sh]: Remove white space here

File: makejdk.sh [hunk 2]
```diff
@@ -199,7 +210,11 @@ if [ "${USE_DOCKER}" ] ; then
     docker ps -a | awk '{ print $1,$2 }' | grep $CONTAINER | awk '{print $1 }' | xargs -I {} docker rm {}
   fi
 
-else
-  echo "Calling sbin/build.sh $WORKING_DIR $TARGET_DIR"
-  ./sbin/build.sh $WORKING_DIR $TARGET_DIR
-fi
+else  
+  echo "Calling sbin/build.sh $WORKING_DIR $TARGET_DIR $BUILD_FULL_NAME"
```
Comment (karianna) [inline makejdk.sh]: Shouldn't the echo match the line below it for full debug?

File: sbin/build.sh (hunk 1 above)
Comment (neomatrix369) [inline sbin/build.sh]: Ok :)

File: makejdk.sh (hunk 2 above)
Comment (neomatrix369) [inline makejdk.sh]: Yeah best such changes or more in next PRs, keeps this PR focused to minimum changes to be merged

**Comment by neomatrix369** [comment]
Will attend to this PR later, let it be hanging here for a bit

File: makejdk.sh (hunk 2 above)
Comment (neomatrix369) [inline makejdk.sh]: Yes will amend asap

**Pull request merged**